apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryClientImpl.InsertData.cs

```diff
@@ -341,5 +321,71 @@ private InsertAllRequest CreateInsertAllRequest(TableReference tableReference, I
             RetryHandler.MarkAsRetriable(request);
             return request;
         }
+
+        /// <summary>
+        /// Wrapper list so that we can be more efficient and reuse the BigQueryInsertRow
+        /// list.
+        /// This list is really read only only the type needs to be IList.
```

Comment:
Nit: only only

(It would be good to say *why* it has to be `IList<T>` - I assume it's because of the type of `TableDataInsertAllRequest.Rows`.